content/js2/blocks/browser/index.md

```diff
@@ -0,0 +1,15 @@
++++
+title = '💻 User interfaces'
+headless = true
+time = 10
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Define static html'
+    2='Explain why we interact with user interfaces'
++++
+
+User interfaces provide the gateway between a user and a complex application.
+When navigating the internet, we continually interact with web pages to access data and interact with complex web applications.
+
+A {{<tooltip title="web browser">}} provides a user interface to interact with web pages.{{</tooltip>}} is capable of fetching and then rendering HTML documents to create a user interface. If a user visits a website and gets a plain HTML document back, we say this content is static. By static, we mean that once the page renders in the browser, it will only change if it is programmed to do so. However, when we interact with the elements on the page, we often expect the page to update.
```

Comment:
I'm not sure introducing a concept of "static" here is particularly useful, or how much depth to go into... Right now this feels quite vague and confusing, I think partially because we don't have anything to contrast it with...

If we want to introduce this here, I'd maybe consider something more like:
```suggestion
A {{<tooltip title="web browser">}} provides a user interface to interact with web pages.{{</tooltip>}} is capable of fetching HTML documents from a server, and then rendering the document to create a user interface. If a user visits a website and gets a plain HTML document back, we say this content is static.

By static, we mean that the server's job was just to hand over the HTML document, and then the browser takes over. A user may interact with the browser's interface, e.g. to scroll, type in a text field, or drag-and-drop an image around, but this is done purely by interacting with the browser - the browser won't talk to the server about this.
```

Reply:
Yeah I struggled with the use of the word **static** here and wasn't satisfied with the introduction I wrote. I think your suggestion makes it a lot clearer :)